doc/sphinx/using/libraries/index.rst

```diff
@@ -16,6 +20,23 @@ the `Coq Package Index <https://coq.inria.fr/opam/www/>`_ and
 installed with the `opam package manager
 <https://coq.inria.fr/opam-using.html>`_.
 
+:gdef:`Libraries <library>` contain only compiled Coq scripts.
+:gdef:`plugins <plugin>` can also include compiled OCaml code that can change
+the behavior of Coq.  Both are :term:`packages <package>`.
+While users configure and load them identically, there are a few differences
+to consider:
+
+- Nearly all plugins add things to the vernacular language that could not be
+  added otherwise, so new top-level commands or tactics are likely available.
+- Plugins can do almost anything to Coq. Therefore they have more potential to
+  expand the trusted code base than libraries, so they shouldn't be fully
+  trusted.  If desired, you can reduce the risk by running :ref:`coqchk`
+  on compiled scripts that load plugins.
+  (`coqchk` doesn't load plugins, so they won't be part of trusted code base.)
+- Plugins that aren't in Coq's CI (continuous integration) system are likely
```

Comment:
If we mention Coq's CI, we should probably add a link to https://github.com/coq/coq/blob/master/dev/ci/README-users.md.

Reply:
Not sure that will be helpful for most users, who probably only want to know what packages are in CI.  Plus we have 3 different identifiers for each CI package that may or may not line up, e.g. `ci-equations`, the associated github web page (perhaps containing documentation) and the module name within Coq.  Maybe worth adding (in another PR) a file in the source tree (which could be automatically checked to some degree) or a list in the doc.

Reply:
This is useful IMHO, not to help users figure out what packages are in CI (although if they really want, they can dig this info from the Coq repo), but so that interested users can tell plugin writers to put their plugin into Coq's CI.

Reply:
OK, added it.